core/threads_files.go

```diff
@@ -43,6 +43,20 @@ type ThreadLikeInfo struct {
 	Username string    `json:"username,omitempty"`
 }
 
+type ThreadJoinInfo struct {
```

Comment:
copy-paste here... do we need these? reuse other interface perhaps?

Reply:
yeah, hmmm. I see what you mean. I just added username to the BlockInfo object, which is available via Textile.BlockInfo(blockId)... how about just replaceing `Block` on ThreadUpdate w/ `BlockInfo`, and then only in the case where you want to inflate the files update info tack on the `Info` field?